debug.sh

```diff
@@ -2,48 +2,69 @@
 trap "" 0 1 2 3 9 13 15
 
 # User configuration
-HOME_DIRECTORY="/home/rhydon"
-BUILDROOT_IMAGES_PATH="$HOME_DIRECTORY/workspace/buildroot-2020.02.4/output/images"
+HOME_DIRECTORY="HOME_DIRECTORY_PLACEHOLDER"
```

Comment:
Create a local gdbinit file and use '-x' option.

Reply:
good idea